docker/deploy/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:xenial
+RUN apt-get update
+RUN apt-get -y install apt-utils
+RUN apt-get -y install sudo
+RUN apt-get -y install git
```

Comment:
Is this `apt-get -y install git` redundant? Since it looks like we get it in the next line. Same question about the other dockerfile.